Description: In absence of client.conf configuration, the default domain socket
 (/var/run/cups/cups.sock) will get checked for read, write and execution bits,
 but the latter doesn't make sense
Author: Didier Raboud <odyx@debian.org>
Origin: vendor
Bug: https://www.cups.org/str.php?L4679
Bug-Debian: https://bugs.debian.org/792730
Last-Update: 2015-07-18
--- a/cups/usersys.c
+++ b/cups/usersys.c
@@ -1017,7 +1017,7 @@
     struct stat	sockinfo;		/* Domain socket information */
 
     if (!stat(CUPS_DEFAULT_DOMAINSOCKET, &sockinfo) &&
-	(sockinfo.st_mode & S_IRWXO) == S_IRWXO)
+	(sockinfo.st_mode & (S_IROTH | S_IWOTH) ) == (S_IROTH | S_IWOTH))
       cups_set_server_name(cc, CUPS_DEFAULT_DOMAINSOCKET);
     else
 #endif /* CUPS_DEFAULT_DOMAINSOCKET */
